ouster_client/include/ouster/client.h

```diff
@@ -174,9 +218,12 @@ bool get_config(const std::string& hostname, sensor_config& config,
 enum config_flags : uint8_t {
     CONFIG_UDP_DEST_AUTO    = (1 << 0), ///< Set udp_dest automatically
     CONFIG_PERSIST          = (1 << 1), ///< Make configuration persistent
-    CONFIG_FORCE_REINIT     = (1 << 2)  ///< Forces the sensor to re-init during
+    CONFIG_FORCE_REINIT     = (1 << 2), ///< Forces the sensor to re-init during
                                         ///< set_config even when config params
                                         ///< have not changed
+    CONFIG_GET_ACTIVE       = (1 << 3)  ///< Get active config from sensor  
```

Comment:
Are you still using this?

Reply:
I didn't observe this before, @ShepelIlya is this really needed?

Reply:
I think that with this we can make initialization of secondary clients more secure. So core idea to read full-config on secondary sensors, without configuring them. And i see 2 ways of doing this - 
1) additional flag that keeps us from configuring
2) change OnInit function, something like that
`class OusterSensor : public OusterClientBase {
   private:
    virtual void onInit() override {
        auto& pnh = getPrivateNodeHandle();
        sensor_hostname = get_sensor_hostname(pnh);
        sensor::sensor_config config;
        u_int8_t flags;
        std::tie(config, flags) = create_sensor_config_rosparams(pnh);
        if (/* maybe we need rosparam here?*/)
             configure_sensor(sensor_hostname, config, flags);
        else
             get_config(sensor_hostname, config, true); 
        sensor_client = create_sensor_client(sensor_hostname, config);
        update_config_and_metadata(*sensor_client);
        save_metadata(pnh);
        OusterClientBase::onInit();
        create_get_config_service();
        create_set_config_service();
        start_connection_loop();
    }`
    
    If you see more correct way to do it  then i can try to implement it.

Reply:
@Samahu Will have to comment on this one again. I was paying attention to the ros nodelet when I made my comment and he knows a lot more about it.